### ISSUE TITLE
"The term 'cinst' is not recognized as the name of..."

### ISSUE THREAD

**Issue by AnOldMan** [KirillOsenkov/CodeCleanupTools#19]
Usage not for newbs apparently!  How do I join the club?
Nowhere in your readme does it tell someone who has just used VS to do Web Dev how to actually compile/build/deploy/use this project.  What is "cinst"?  What prompt am I using?  I run "SortProjectItems" it opens command prompt with a prompt to press any key to close.  CMD & PS don't recognize cinst or SortProjectItems, whether opened from VS or not.

It would be awesome if you could reveal these mysteries to the un-initiated! Thanks!

**Comment by KirillOsenkov** [comment]
Yes, these are raw and some assembly is certainly required, sorry about that.

This project is a collection of tools for various purposes, did you have a particular tool in mind? I put it out there to save people time so they can just adapt these to their needs instead of writing stuff from scratch.

after you `git clone`, you can go to Start Menu -> Developer Command Prompt for VS 2019, then cd to the directory which you just cloned (I assume C:\CodeCleanupTools), then type `msbuild /r /m /bl` to build it.

After that you could run SortProjectItems for instance using:

`dotnet SortProjectItems\bin\Debug\netcoreapp2.0\SortProjectItems.dll /?`

This will print help.

But honestly, if you don't know what this tool is for, you probably don't need it. In the last few years the new "SDK-style" .csproj file format became popular, and it doesn't have the problem that SortProjectItems is solving.

**Comment by KirillOsenkov** [comment]
`cinst` is Chocolatey installer, https://chocolatey.org is a way to quickly install software, you'd need to install Chocolatey first via https://chocolatey.org/install, then cinst would work.

But the instructions above don't require Chocolatey, so you can build it yourself.

If you tell me what problem you're looking to solve, I may be able to give more detailed instructions on how to do that. Thanks!

**Comment by AnOldMan** [comment]
Thanks!
Older project with years of messy content includes, Googled for sorting tool this seemed perfect.  Building the solution created the dlls in bin\Debug already.  Now I know how to invoke from prompt, will give it a go.  Have src control so no worries about mess-ups.
